palindrome/lib/palindrome.ex

```diff
@@ -1,14 +1,15 @@
 defmodule Palindrome do
   @moduledoc false
 
-  # TODO test for valid false input
   def get_palindrome(input) when is_integer(input) do
     input
     |> Integer.to_string()
-    |> get_palindrome()
+    |> process_input()
```

Comment:
process_input name is extremely generic, not creating clarity on what it does. process_input may mean anything.